src/connectedk8s/azext_connectedk8s/azext_metadata.json

```diff
@@ -1,6 +1,5 @@
 {
     "name": "connectedk8s",
     "version": "1.2.1",
-    "azext.minCliCoreVersion": "2.16.0",
-    "azext.maxCliCoreVersion": "2.29.0"
+    "azext.minCliCoreVersion": "2.16.0"
```

Comment:
The modification here is because migrating `MSAL` can be compatible with the higher version of `azure-cli-core`, right?

Reply:
Yes that's right.